widget/view.controller.js

```diff
@@ -2,38 +2,186 @@
   MIT License
   Copyright (c) 2024 Fortinet Inc
   Copyright end */
-  
+
 'use strict';
 (function () {
-    angular
-      .module('cybersponse')
-      .controller('soarFrameworkConfigurationWizard100Ctrl', soarFrameworkConfigurationWizard100Ctrl);
-
-    soarFrameworkConfigurationWizard100Ctrl.$inject = ['$scope', 'widgetUtilityService', '$rootScope', 'widgetBasePath', 'WizardHandler'];
-
-    function soarFrameworkConfigurationWizard100Ctrl($scope, widgetUtilityService, $rootScope, widgetBasePath, WizardHandler) {
-      $scope.moveNext = moveNext;
-      $scope.isLightTheme = $rootScope.theme.id === 'light';
-      $scope.startPageImage = $scope.isLightTheme ? widgetBasePath +'images/sfspStartLight.png': widgetBasePath +'images/sfspStartDark.png';
-
-      function _handleTranslations() {
-        widgetUtilityService.checkTranslationMode($scope.$parent.model.type).then(function () {
-          $scope.viewWidgetVars = {
-            // Create your translating static string variables here
-          };
-        });
+  angular
+    .module('cybersponse')
+    .controller('soarFrameworkConfigurationWizard100Ctrl', soarFrameworkConfigurationWizard100Ctrl);
+
+  soarFrameworkConfigurationWizard100Ctrl.$inject = ['$scope', 'widgetUtilityService', '$rootScope', 'widgetBasePath', 'WizardHandler', 'soarConfigService', 'toaster'];
+
+  function soarFrameworkConfigurationWizard100Ctrl($scope, widgetUtilityService, $rootScope, widgetBasePath, WizardHandler, soarConfigService, toaster) {
+    $scope.gblVarToKeyStoreMapping = {
+      "Excludelist_IPs": {
+        "keystore": "sfsp-excludelist-ips",
+        "defaultValue": "8.8.8.8,10.1.1.2"
+      },
+      "Excludelist_URLs": {
+        "keystore": "sfsp-excludelist-urls",
+        "defaultValue": "https://www.google.com,https://mail.yahoo.com/login.html,https://www.office.com/"
+      },
+      "Excludelist_Domains": {
+        "keystore": "sfsp-excludelist-domains",
+        "defaultValue": "google.com,yahoo.com,fortinet.net,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com,fortiguard.com,forticloud.com,w3.org"
+      },
+      "Excludelist_Files": {
+        "keystore": "sfsp-excludelist-files",
+        "defaultValue": ""
+      },
+      "Excludelist_Ports": {
+        "keystore": "sfsp-excludelist-ports",
+        "defaultValue": ""
+      },
+      "CIDR_Range": {
+        "keystore": "sfsp-cidr-range",
+        "defaultValue": "10.0.0.0/8,172.16.0.0/12,192.168.0.0/16"
+      }
```

Comment:
add this to constants in service layer, and avoid using $scope for variables not used in html, use var instead

Reply:
DONE